cms/categories/templates/categories/category_detail.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-basepage{% endblock body_class %}
+
+{% block content %}
+
+<div class="nhsuk-width-container">
+    <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+            <h1>{{ object.name }}</h1>
+        </div>
+    </div>
+
+    <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+          <p>{{ object.description }}</p>
+          {% if category_pages %}
+            <table role="table" class="nhsuk-table-responsive">
+              <thead class="nhsuk-table__head">
+                <tr>
+                  <th>Title</th>
+                  <th>Content type</th>
+                  <th>Date published/updated</th>
```

Comment:
This feels unnecessarily ambiguous; would "Last updated" work better?

Reply:
This is the wording from NHS. If we get a later change request it's an easy fix.